pkgs/development/python-modules/ftputil/default.nix

```diff
@@ -18,12 +18,15 @@ buildPythonPackage rec {
     py.test test \
       -k "not test_public_servers and not test_real_ftp \
           and not test_set_parser and not test_repr \
-          and not test_conditional_upload and not test_conditional_download_with_older_target"
-  '';
+          and not test_conditional_upload and not test_conditional_download_with_older_target \
+  ''
+  # need until https://ftputil.sschwarzer.net/trac/ticket/140#ticket is fixed
+  + stdenv.lib.optionalString stdenv.isDarwin ''and not test_error_message_reuse''
```

Comment:
I am not sure how to write this more elegant but it works.

Reply:
Import both stdenv and lib, so you can use the lib shorthand where appropriate. Other than that lgtm.

Reply:
Think I solved that.